domains/bernardofoegbu.json

```diff
@@ -6,6 +6,6 @@
     "email": "obinnaofoegbu71@gmail.com"
   },
   "record": {
-    "CNAME": "o-bernardofoegbu.github.io"
+     "TXT": "google-site-verification=VAoiuOjvQxUCsqxyZUGS9WqMLVi2lSxOAnmyENy0Sq4"
```

Comment:
Should mention this. TXT records are used for verification, but when they're not paired with any other records, they're pretty useless. You should add the other records that Google requires.